rpc/block_svc_routines.c

```diff
@@ -5015,11 +5019,12 @@ block_list_cli_1_svc_st(blockListCli *blk, struct svc_req *rqstp)
 
   tgmdfd = glfs_opendir (glfs, GB_METADIR);
   if (!tgmdfd) {
-    errCode = errno;
+    unsigned int errorCode = errno;
     GB_ASPRINTF (&errMsg, "Not able to open metadata directory for volume "
-                 "%s[%s]", blk->volume, strerror(errCode));
+                 "%s[%s]", blk->volume, strerror(errorCode));
     LOG("mgmt", GB_LOG_ERROR, "glfs_opendir(%s): on volume %s failed[%s]",
-        GB_METADIR, blk->volume, strerror(errCode));
+        GB_METADIR, blk->volume, strerror(errorCode));
+    errCode = errno;
```

Comment:
On a last minute walk through noticed, this should have been 'errCode = errorCode;' instead of errno. All other things look good. Will resubmit (along with Reviewed-By: flag)

Reply:
Done